src/config/backends/extlinux.ts

```diff
@@ -0,0 +1,262 @@
+		);
+
+		// Add config values to `APPEND` directive
+		defaultEntry.APPEND = this.appendDirective.value(
+			appendOptions,
+			defaultEntry.APPEND,
+		);
+
+		// Write new extlinux configuration
+		return await remountAndWriteAtomic(
+			ExtlinuxConfigBackend.bootConfigPath,
+			ExtlinuxConfigBackend.extlinuxFileToString(parsedBootFile),
+		);
+	}
+
+	public isSupportedConfig(configName: string): boolean {
+		return _.includes(ExtlinuxConfigBackend.supportedConfigValues, configName);
+	}
+
+	public isBootConfigVar(envVar: string): boolean {
+		return _.startsWith(envVar, ExtlinuxConfigBackend.bootConfigVarPrefix);
+	}
+
+	public processConfigVarName(envVar: string): string {
+		return envVar.replace(ExtlinuxConfigBackend.bootConfigVarRegex, '$2');
+	}
+
+	public processConfigVarValue(_key: string, value: string): string {
+		return value;
+	}
+
+	public createConfigVarName(configName: string): string {
+		return `${ExtlinuxConfigBackend.bootConfigVarPrefix}${configName}`;
+	}
+
+	private isDirective(configName: string): boolean {
+		return _.includes(
+			ExtlinuxConfigBackend.supportedDirectives,
+			configName.toUpperCase(),
+		);
+	}
+
+	private static parseExtlinuxFile(confStr: string): ExtlinuxFile {
+		const file: ExtlinuxFile = {
+			globals: {},
+			labels: {},
+		};
+
+		// Firstly split by line and filter any comments and empty lines
+		let lines = confStr.split(/\r?\n/);
+		lines = _.filter(lines, (l) => {
+			const trimmed = _.trimStart(l);
+			return trimmed !== '' && !_.startsWith(trimmed, '#');
+		});
+
+		let lastLabel = '';
+
+		for (const line of lines) {
+			const match = line.match(/^\s*(\w+)\s?(.*)$/);
+			if (match == null) {
+				log.warn(`Could not read extlinux entry: ${line}`);
+				continue;
+			}
+			let directive = match[1].toUpperCase();
+			let value = match[2];
+
+			// Special handling for the MENU directive
+			if (directive === 'MENU') {
+				const parts = value.split(' ');
+				directive = `MENU ${parts[0]}`;
+				value = parts.slice(1).join(' ');
+			}
+
+			if (directive !== 'LABEL') {
+				if (lastLabel === '') {
+					// Global options
+					file.globals[directive] = value;
+				} else {
+					// Label specific options
+					file.labels[lastLabel][directive] = value;
+				}
+			} else {
+				lastLabel = value;
+				file.labels[lastLabel] = {};
+			}
+		}
+
+		return file;
+	}
+
+	private static extlinuxFileToString(file: ExtlinuxFile): string {
+		let ret = '';
+		_.each(file.globals, (value, directive) => {
+			ret += `${directive} ${value}\n`;
+		});
+		_.each(file.labels, (directives, key) => {
+			ret += `LABEL ${key}\n`;
+			_.each(directives, (value, directive) => {
+				ret += `${directive} ${value}\n`;
+			});
+		});
+		return ret;
+	}
+
+	private static findDefaultLabel(file: ExtlinuxFile): string {
+		const defaultLabel = _.find(file.globals, (_v, l) => {
+			if (l === 'DEFAULT') {
+				return true;
+			}
+			return false;
+		});
```

Comment:
I think this can be
```suggestion
		const defaultLabel = file.globals.DEFAULT;
```
?

Reply:
Yes, and I was able to make it even simpler by just check if the value is falsey else return it. No need to find.